### DISCUSSION TITLE
Error after exporting from blender

### DISCUSSION THREAD

**Discussion by luisparravicini** [SolarLune/tetra3d#15] [Q&A] [answered]
Hi, I'm getting an error after exporting with the Blender addon. Using the lighting example:

- Open _lighting.blend_ (I'm using Blender 3.2.0)
- Go to the addon UI and click on "Tetra3D Export" (exported file: [lighting.zip](https://github.com/SolarLune/Tetra3d/files/8936519/lighting.zip))
- Run the example (`go run .`). The output is:
 
```
panic: interface conversion: tetra3d.INode is *tetra3d.PointLight, not *tetra3d.Node

goroutine 12 [running]:
main.(*Game).Update(0xc00052e0c0)
	/home/xrm0/dev/a/Tetra3d/examples/lighting/main.go:96 +0xdfc
github.com/hajimehoshi/ebiten/v2.(*imageDumper).update(0xc000e2a190)
	/home/xrm0/go/pkg/mod/github.com/hajimehoshi/ebiten/v2@v2.3.0-alpha.7/imagedumper_desktop.go:111 +0x52
```

The line where the error occurs is:
```go
        light := g.Scene.Root.Get("Point light").(*tetra3d.Node)
```

Is there something I should have done before exporting and because of that this error happens or is it a bug?

The only modification done in the example is to have exported the blender file.

**Comment by SolarLune** [comment, marked as answer]
Whoops, my bad, I never noticed this. This should be fixed with the latest lighting example; the core issue is that it's a point light, not a Node, so I've updated the lighting example with both a fresh GLTF export and a fixed main.go file. I've recreated the v0.8.1 tag/release with this fix. Thanks for the report~!
